code/src/pages/product-details/components/product-details-container/ProductDetailsContainer.test.tsx

```diff
@@ -164,11 +164,14 @@ describe("ProductDetailsContainer", () => {
 
   test("renders description correctly", () => {
     renderAndMock({
-      data: { ...mockProduct, description: "value1/value2/value3/value4" }
+      data: mockProduct
     });
 
-    const descriptionParagraphs = screen.getAllByText(/value\d/);
-    expect(descriptionParagraphs).toHaveLength(4);
+    mockProduct.description.split("/").forEach((item) => {
+      const regex = new RegExp(item.replace(/[.*+?^${}()|[\]\\]/g, "\\$&"));
```

Comment:
What is the role of this regular expression? Can you explain how it works? Let's try to simplify it and move to `regexes.ts` as you did with UUID before